waku/v2/node/wakunode2.nim

```diff
@@ -74,6 +75,7 @@ type
     wakuRlnRelay*: WakuRLNRelay
     wakuLightPush*: WakuLightPush
     peerInfo*: PeerInfo
+    enr*: Record
```

Comment:
enr seems a bit short, and Record a bit generic. Can we make this more descriptive somehow?